rest/model/task.go

```diff
@@ -16,8 +16,9 @@ import (
 )
 
 const (
-	TaskLogLinkFormat  = "%s/task_log_raw/%s/%d?type=%s"
-	EventLogLinkFormat = "%s/event_log/task/%s"
+	TaskLogLinkFormat        = "%s/task_log_raw/%s/%d?type=%s"
+	ParsleyTaskLogLinkFormat = "%s/evergreen/%s/%d/%s" // URL,taskId, execution, type
```

Comment:
it looks like the comment is applicable to the taskloglinkformat so can we put it above?

Reply:
this was actually just for my own thought process so i'll just pull it out